### PR TITLE
Fix links, remove overview file, add light push

### PR DESCRIPTION
**Pull request by D4nte**
Overview did not really add much value as it gave slightly more than the quick start.
Better direct the developers to the guides once quick start is done.

### CODE REVIEW

**Review by jm-clius — APPROVED** [review]
LGTM

File: src/guides/light_push_send_messages.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+# Send Messages Using Waku Light Push
+
+Waku Light Push enables a client to receive a confirmation when sending a message.
+
+The Waku Relay protocol sends messages to connected peers but does not provide any information on whether said peers have received messages.
+This can be an issue when facing potential connectivity issues.
```
Comment (oskarth) [inline src/guides/light_push_send_messages.md:6]: For example, when peer count is lower than <some number> (gossipsub low parameter probably, 4?)

File: src/introduction.md [hunk 2]
```diff
@@ -2,22 +2,24 @@
...
 built for browser environment.
 
-The [guides](./guides) describe specific usages of js-waku in more details.
+The [quick start](./quick_start.md) presents an easy way to send and receive messages using js-waku.
 
-The js-waku repository also holds a number of [examples](./examples.md).
-The examples are working proof-of-concepts that demonstrate several features of Waku.
-Check out the [example list](./examples.md) to see what feature each example demonstrates. 
+The [guides](./guides) explain specific js-waku features
+and how it can be used with popular web frameworks.
```
Comment (oskarth) [inline src/introduction.md:18]: This reminds me, DigitalOcean have a wiki and they did "content farming". Basically they paid bounties to technical writers to write tutorial like "How to setup Nginx on Ubuntu 15.10" <repeat for 100s of permutations>. This is useful for people to copy paste, and also works as a form of SEO.

Perhaps we can do a light version here with bounties for React/Vue etc?

**Review by oskarth — APPROVED** [review]

File: src/introduction.md (hunk 2 above)
Comment (D4nte) [reply src/introduction.md:18]: this is a good idea. In line with my mention of retrpactive funding for the Vue.js post.

Will add that to my task list :)

**Pull request merged**